src/django/pfb_analysis/models.py

```diff
@@ -184,33 +184,42 @@ def create_environment(**kwargs):
             logger.warn('Attempt to re-run job: {}. Skipping.'.format(self.uuid))
             return
 
-        client = boto3.client('batch')
-
-        # Since we run django manage commands in the analysis container, it needs a copy of
-        # all the environment variables that this app needs, most of which are conveniently
-        # prefixed with 'PFB_'
-        # Set these first so they can be overridden by job specific settings below
```

Comment:
Good point. I don't know if the PFB variables coming into the Django container from the environment are likely to end up needing overrides, but certainly we shouldn't clobber them if they do...